components/Fly.js

```diff
@@ -25,7 +24,7 @@ class Fly extends Component{
           onPress: () => console.log("Cancel Pressed"),
           style: "cancel"
         },
-        { text: "OK", onPress: () => deleteFly(flyId) }
+        { text: "OK", onPress: () => this.props.handleDelete(flyId) }
```

Comment:
I moved this to be something the parent component, FlyBox, handles due to issues with updating the page